hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/replication/GrpcReplicationService.java

```diff
@@ -157,11 +156,7 @@ public void download(CopyContainerRequestProto request,
       IOUtils.cleanupWithLogger(LOG, outputStream);
 
       if (copyContainerZeroCopyMessageMarshaller != null) {
```

Comment:
I think we can also remove `zeroCopyEnabled` here.

Reply:
Thank you for the reminder